inst/templates/vignette.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "{{{ vignette_title }}}"
+author: "{{{ vignette_author }}}"
```

Comment:
I think we should just remove the `author` and `date` fields altogether. I don't think they're very useful.